providers/inline-evaluating-provider/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>dev.openfeature.contrib</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.1.0</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <!-- The group id MUST start with dev.openfeature, or publishing will fail. OpenFeature has verified ownership of this (reversed) domain. -->
+    <groupId>dev.openfeature.contrib.providers</groupId>
+    <artifactId>inline-evaluating-provider</artifactId>
+    <version>0.0.1</version> <!--x-release-please-version -->
+
+    <name>inline-evaluating-provider</name>
```

Comment:
can we rename it as `jsonlogic-eval-provider`?  when we say inline it's more generic and I feel even `env-var`, `in-memory-eval` all  falls into this category?